src/test/lib/storage/fixed_string_dictionary_segment/fixed_string_vector_test.cpp

```diff
@@ -106,6 +106,15 @@ TEST_F(FixedStringVectorTest, Erase) {
   EXPECT_EQ((*fixed_string_vector)[0u], "foo");
 }
 
+TEST_F(FixedStringVectorTest, EraseEmptyVector) {
+    std::vector<pmr_string> v1 = {};
+    auto empty_fixed_string_vector = FixedStringVector{v1.begin(), v1.end(), 0};
+    EXPECT_EQ(empty_fixed_string_vector.size(), 0u);
+
+    auto it = empty_fixed_string_vector.begin();
+    EXPECT_NO_THROW(empty_fixed_string_vector.erase(it, empty_fixed_string_vector.end()));
```

Comment:
maybe add a check that size still is `0` afterwards? Then, feel free to remove the `EXPECT_NO_THROW` wrapping the `erase` call.